scripts/x-data-grid.exports.json

```diff
@@ -270,6 +270,7 @@
   { "name": "GridPreferencesPanel", "kind": "Variable" },
   { "name": "GridPreferencesPanelApi", "kind": "Interface" },
   { "name": "GridPreProcessEditCellProps", "kind": "Interface" },
+  { "name": "GridPreProcessingGroupLookup", "kind": "ExportSpecifier" },
```

Comment:
I need to export it to be able to extend it.
I'll try to only export it from a sub module not from the root